source/material_model/grain_size.cc

```diff
@@ -1064,6 +1109,12 @@ namespace aspect
                              "The grain size $d_{ph}$ to that a phase will be reduced to when crossing a phase transition. "
                              "When set to zero, grain size will not be reduced. "
                              "Units: \\si{\\meter}.");
+          prm.declare_entry ("Volume fraction of phase", "0.4",
+                             Patterns::List (Patterns::Double (0., 1.)),
```

Comment:
Why is this a list of doubles rather than just a double? From the description, it sounds like it would be just one number.

Reply:
You are right, it should be only double since we decided on using only one phase with the pinned state formulation.